.github/workflows/Format.yml

```diff
@@ -1,42 +1,41 @@
+---
 name: Format Check
 on:
-  # because we rely on ReviewDog to trigger failure, there's no reason to run on anything
+  push:
+    branches:
+      - main
+      - /^release-.*$/
+    tags: ["*"]
+    paths:
+      - "**/*.jl"
+      - ".github/workflows/Format.yml"
   pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
     paths:
       - "**/*.jl"
-      - ".github/workflows/FormatCheck.yml"
-concurrency:
-  # Skip intermediate builds: always.
-  # Cancel intermediate builds: only if it is a pull request build.
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+      - ".github/workflows/Format.yml"
 jobs:
   format-check:
-    if: ${{ github.event_name == 'pull_request' && github.event.pull_request.draft == false }}
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        version:
-          - "1.9"
-        os:
-          - ubuntu-latest
-        arch:
-          - x64
+    name: Julia
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@v2
         with:
-          version: ${{ matrix.version }}
-          arch: ${{ matrix.arch }}
+          version: "1"
       - uses: julia-actions/cache@v2
-        with:
-          cache-name: "${{ github.workflow }}-${{ github.job }}-${{ matrix.pkg.name }}-${{ matrix.version }}-${{ matrix.os }}-${{ matrix.arch }}"
-          cache-compiled: true
-      - name: Instantiate `format` environment and format
+      - name: Install JuliaFormatter
+        shell: julia --project=@format --color=yes {0}
```

Comment:
Did you mean to delete the contents of the `format/` directory? I think this creates a new `format/Project.toml`, which we aren't `gitignore`ing.

Reply:
It uses a shared environment, not a local one (note the `@`)